.env.sample

```diff
@@ -6,4 +6,7 @@ A_PASSWORD="<priv_password>"
 B_EMAIL="<reg_email>"
 B_PASSWORD="<reg_password>"
 
-CATEGORY="all"
\ No newline at end of file
+CATEGORY="plugin"
+
+# set to true if whathost debugging is needed
```

Comment:
Suggest commented example of output if true. 

Also: I don't really know what "whathost" means so some small better description of why to set true or false would really be helpful.